### PR TITLE
Fixup ctrl-F crash

### PR DESCRIPTION
**Pull request by jonko0493**
Also add a small little authors section to the README as well.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,6 +25,18 @@
   <img width="325px" src="https://haroohie.club/images/chokuretsu/serial-loops/home-screen.png" alt="Screenshot of the Serial Loops home screen. The Serial Loops logo and title sits at the top of the menu. Below that, under 'Start' on the left hand side, options to create a project, open an existing project, and modify preferences are present. An empty list of 'Recents' is visible on the right hand side, where recent projects would appear." />
 </p>
 
+## Authors
+### Developers
+Serial Loops is developed by:
+* [Jonko](https://github.com/jonko0493) &ndash; Systems architect & reverse engineering work
+* [William278](https://william278.net) &ndash; UX architect & design work
+
+### Translators
```
Comment (WiIIiam278) [inline README.md:34]: Perhaps we should add these to the about screen?

File: src/SerialLoops/SerialLoops.csproj [hunk 2]
```diff
@@ -27,7 +27,7 @@
     <None Remove="Icons\Bottom_Screen.png" />
     <None Remove="Icons\Camera.png" />
     <None Remove="Icons\Character_Sprite.png" />
-    <None Remove="Icons\Chess.png" />
+    <None Remove="Icons\Chess_Puzzle.png" />
```
Comment (WiIIiam278) [inline src/SerialLoops/SerialLoops.csproj:30]: whoops

**Review by WiIIiam278 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (jonko0493) [reply README.md:34]: Yes, that's my plan once I figure out how to make it work :)

File: src/SerialLoops/SerialLoops.csproj (hunk 2 above)
Comment (jonko0493) [reply src/SerialLoops/SerialLoops.csproj:30]: My bad for sure

**Pull request merged**